test/reflect.js

```diff
@@ -0,0 +1,483 @@
+  });
+
+  describe('Reflect.getPrototypeOf()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.getPrototypeOf).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.getPrototypeOf(item);
+      });
+    });
+  });
+
+  describe('Reflect.has()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.has).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.has(item, 'prop');
+      });
+    });
+
+    it('will detect own properties', function () {
+      var target = Object.create(null);
+
+      expect(Reflect.has(target, 'prop')).to.equal(false);
+
+      target.prop = undefined;
+      expect(Reflect.has(target, 'prop')).to.equal(true);
+
+      delete target.prop;
+      expect(Reflect.has(target, 'prop')).to.equal(false);
+
+      Object.defineProperty(target, 'accessor', {
+        set: function () {}
+      });
+
+      expect(Reflect.has(target, 'accessor')).to.equal(true);
+
+      expect(Reflect.has(Reflect.has, 'length')).to.equal(true);
+    });
+
+    it('will search the prototype chain', function () {
+      var intermediate = Object.create(object),
+      target = Object.create(intermediate);
+
+      intermediate.some_property = undefined;
+
+      expect(Reflect.has(target, 'bool')).to.equal(true);
+      expect(Reflect.has(target, 'some_property')).to.equal(true);
+      expect(Reflect.has(target, 'undefined_property')).to.equal(false);
+    });
+  });
+
+  describe('Reflect.isExtensible()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.isExtensible).to.equal('function');
+    });
+
+    it('returns true for plain objects', function () {
+      expect(Reflect.isExtensible({})).to.equal(true);
+      expect(Reflect.isExtensible(Object.preventExtensions({}))).to.equal(false);
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.isExtensible(item);
+      });
+    });
+  });
+
+  describe('Reflect.ownKeys()', function () {
+    it('is a function', function () {
+      expect(typeof Reflect.ownKeys).to.equal('function');
+    });
+
+    it('throws if the target isn\'t an object', function () {
+      testPrimitiveThrow(function (item) {
+        return Reflect.ownKeys(item);
+      });
+    });
+
+    it('should return the same result as Object.getOwnPropertyNames if there are no Symbols', function () {
+      var obj = { foo: 1, bar: 2 };
+
+      obj[1] = 'first';
+
+      var result = Object.getOwnPropertyNames(obj);
+
+      expect(Reflect.ownKeys(obj)).to.deep.equal(result);
+      expect(result).to.deep.equal(['1', 'foo', 'bar']);
+    });
+
+    (typeof Symbol === 'function' ? it : xit)('symbols come last', function () {
```

Comment:
Let's make a var at the top of the file, something like `var ifSymbolsIt = typeof Symbol === 'function' ? it : xit` so that multiple tests can cleanly do `ifSymbolsIt('does something', …)`